homepage/homepage/content/docs/groups/sharing.mdx

```diff
@@ -92,3 +92,236 @@ useAcceptInvite({
 });
 ```
 </CodeGroup>
+
+
+### Requesting Invites
+
+To allow a non-group member to request an invitation to a group you can use the `writeOnly` role.
+This means that users only have write access to a specific requests list (they can't read other requests). 
+However, Administrators can review and approve these requests.
+
+First, your data models
+
+<CodeGroup>
+```ts twoslash
+// @errors: 2345;
+import { CoMap, co, CoValue } from "jazz-tools";
+
+class Request extends CoMap {
+  // Add properties as needed for your Request class
+  message = co.string;
+  status = co.string;
+}
+
+export class RequestsList extends CoMap.Record(co.ref(Request)) {};
+
+// ---cut-before---
+class RequestsToJoin extends CoMap {
+  writeOnlyInvite = co.string;  // Stores the write-only invite link
+  requests = co.ref(RequestsList);  // Reference to all requests
+}
+```
+</CodeGroup>
+
+Next, set up the request system with appropriate access controls
+
+<CodeGroup>
+```ts twoslash
+import { Group, co, CoList, CoMap, Account } from "jazz-tools";
+import { createInviteLink } from "jazz-react";
+
+class Project extends CoMap {
+  title = co.string;
+  account = co.ref(Account);
+}
+
+class Request extends CoMap {
+  message = co.string;
+  status = co.string;
+}
+
+export class RequestsList extends CoMap.Record(co.ref(Request)) {};
+
+export class RequestsToJoin extends CoMap {
+  writeOnlyInvite = co.string;
+  requests = co.ref(RequestsList);
+}
+
+// ---cut-before---
+function createRequestsToJoin() {
+  const everyoneReaderGroup = Group.create();
+  everyoneReaderGroup.addMember("everyone", "reader");
+
+  const requestsGroup = Group.create();
+  const writeOnlyInvite = createInviteLink(requestsGroup, "writeOnly");
+
+  return RequestsToJoin.create({ 
+    writeOnlyInvite, 
+    requests: RequestsList.create({}, requestsGroup)
+  }, everyoneReaderGroup);
+}
+```
+</CodeGroup>
+
+Using the write-only invitation link, users can submit requests.
+
+<CodeGroup>
+```ts twoslash
+// @errors: 2554 2559
+import { co, CoList, CoMap, ID, Account, CoValue, CoValueClass, Group } from "jazz-tools";
+import { createInviteLink } from "jazz-react";
+
+type JsonValue = string | number | boolean | null | { [key: string]: JsonValue } | JsonValue[];
+type JsonObject = { [key: string]: JsonValue };
+type RawCoMap<T extends Record<string, any>, M extends JsonObject | null> = {
+  _raw: { data: T; metadata: M };
+};
+
+class Project extends CoMap {
+  title = co.string;
+  account = co.ref(Account);
+}
+
+class Request extends CoMap {
```

Comment:
`Request` may clash with the global `Request` type, making automatic imports of IDEs more "painful".

How about renaming this to `JoinRequest`?